docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -0,0 +1,192 @@
+- `SLIDE` the drawer will appear attached to the content view, opening it slides both the drawer and the content view.
+
+| `FRONT`                                               | `BACK`                                               | `SLIDE`                                               |
+| ----------------------------------------------------- | ---------------------------------------------------- | ----------------------------------------------------- |
+| <img src={useBaseUrl('gifs/new-drawer-front.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-back.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-slide.gif')} /> |
+
+### `edgeWidth`
+
+width of the invisible draggable area on the edge of the content view, which can be dragged to open the drawer.
+
+### `hideStatusBar`
+
+a boolean value, when set to `true`, drawer component will use [StatusBar API](http://reactnative.dev/docs/statusbar.html) to hide the OS status bar when the drawer is dragged or idle in the `open` position.
+
+### `statusBarAnimation`
+
+a string, possible values are: `slide`, `none` or `fade`, defaults to `slide`.
+May be used in combination with `hideStatusBar` to select the animation used for hiding the status bar.
+See [StatusBar API](http://reactnative.dev/docs/statusbar.html#statusbaranimation) docs.
+
+### `overlayColor`
+
+color (default `rgba(0, 0, 0, 0.7)`) of the background overlay on top of the content window when the drawer is `open`.
+This color animates from `0%` to `100%` opacity as drawer animates from `closed` to `open`.
+
+### `renderNavigationView`
+
+a renderer function for the drawer componenet.
+Provided with a `progress` SharedValue parameter that indicates the progress of drawer opening/closing. Equals `0` when drawer is `closed` and `1` when `open`.
+This can be used by the drawer component to animated its children while the drawer is opening or closing.
+
+### `onDrawerClose`
+
+a function which is called when the drawer is closed.
+
+### `onDrawerOpen`
+
+a function which is called when the drawer is opened.
+
+### `onDrawerSlide`
+
+function. Called when drawer is moving or animating.
+Provided with a `progress` SharedValue parameter that indicates the progress of drawer opening/closing. Equals `0` when drawer is `closed` and `1` when `open`.
+
+### `onDrawerStateChanged`
+
+a function which is called when the status of the drawer changes. It takes two arguments:
+
+- `newState` - interaction state of the drawer. It can be one of the following:
+  - `DrawerState.IDLE`
+  - `DrawerState.DRAGGING`
+  - `DrawerState.SETTLING`
+- `drawerWillShow` - `true` when `drawer` started animating to `open` position, `false` otherwise.
+
+### `enableTrackpadTwoFingerGesture` (iOS only)
+
+enables two-finger gestures on supported devices, for example iPads with trackpads.
+If not enabled the gesture will require click + drag, with enableTrackpadTwoFingerGesture swiping with two fingers will also trigger the gesture.
```

Comment:
```suggestion
If not enabled the gesture will require click + drag, with `enableTrackpadTwoFingerGesture` swiping with two fingers will also trigger the gesture.
```

Reply:
fixed in [060d2da](https://github.com/software-mansion/react-native-gesture-handler/pull/3188/commits/060d2da676256ed4a08d517427bc93327bb76e2d)